README.md

```diff
@@ -47,187 +11,124 @@ After you're finished please remove all the comments and instructions!
+- **Added migrations for the database**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🚀 Video Demo <a name="https://www.loom.com/share/f851077fcf24473f98f37a8bbeb42a1f"></a>
+
+## 🚀 Live Demo <a name="https://budget-app.herokuapp.com/"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
-In order to run this project you need:
+In order to run this project you need Ruby and  Postgresql installed on your computer.
 
-<!--
-Example command:
+### Setup
 
-```sh
- gem install rails
-```
- -->
+Clone this repository to your desired folder: 
 
-### Setup
+`git clone https://github.com/Estherstarr/Budget-App.git`
+ cd Budget-App
 
-Clone this repository to your desired folder:
+### Install
 
-<!--
-Example commands:
+Then install the project's necessary dependencies with: 
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+`bundle install`
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  Run bundle install
+  Run rails db:create
+  Run rails db:migrate
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  ./bin/dev
+  rails server or rails s
 ```
---->
+
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
 
 ```sh
-  bin/rails test test/models/article_test.rb
+  To Run all tests - rspec spec --format documentation
+  To Run model tests - rspec spec/models
+  To Run integration tests - rspec spec/system
 ```
---->
-
-### Deployment
-
-You can deploy this project using:
 
-<!--
-Example:
-
-```sh
 
-```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **onwuanibe Onomeh** 
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/Estherstarr/)
+- Twitter: [@twitterhandle](https://twitter.com/AnibeEsther)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/onwuanibeonome)
 
```

Comment:
- [ ] `Optional`
- [ ] Kindly add your `social handles`.
- [ ] Its important for employers and other developers when they want to find you on social accounts. 🥂